test/e2e/clusterctl_upgrade.go

```diff
@@ -612,43 +612,26 @@ func ClusterctlUpgradeSpec(ctx context.Context, inputGetter func() ClusterctlUpg
 				input.PostUpgrade(managementClusterProxy, workloadClusterNamespace, managementClusterName)
 			}
 
-			// After the upgrade check that MachineList is available. This ensures the APIServer is serving without
-			// error before checking that it `Consistently` returns the MachineList later on.
-			Byf("[%d] Waiting for MachineList to be available", i)
+			Byf("[%d] Waiting for three minutes before checking if an unexpected rollout happened", i)
```

Comment:
@sbueringer : not sure if we should have an eventually also before the sleep to check it is green at least once. Otherwise we may run into the issue that a rollout may happen, but did not yet because controllers were blocked too?

Reply:
Sounds good